src/duckstation-qt/achievementlogindialog.ui

```diff
@@ -40,24 +34,17 @@
         <string/>
        </property>
        <property name="pixmap">
-        <pixmap resource="resources/resources.qrc">:/icons/emblem-person-blue.png</pixmap>
+        <pixmap resource="resources/duckstation-qt.qrc">:/icons/emblem-person-blue.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
-       <property name="font">
-        <font>
-         <pointsize>14</pointsize>
-         <weight>50</weight>
-         <bold>false</bold>
-        </font>
```

Comment:
This is not working on macOS (has no effect) so I've replaced it with Markdown text. Dunno if there's a better way... maybe the font size can be set programmatically? Also I have no idea how this looks on Windows (it's fine on Linux and macOS)

Reply:
The current setup works fine on Windows.. might be a uic issue and need to manually set the font.